Modules/Quotas/test/Quotas.Application.Tests/Tests/Repositories/MessagesRepositoryTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Backbone.Modules.Messages.Domain.Entities;
+using Backbone.Modules.Messages.Infrastructure.Persistence.Database;
+using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Infrastructure.Persistence.Database;
+using Backbone.Modules.Quotas.Infrastructure.Persistence.Repository;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using Enmeshed.Tooling;
+using Enmeshed.UnitTestTools.Data;
+using Enmeshed.UnitTestTools.TestDoubles.Fakes;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit;
+using MessageEntity = Backbone.Modules.Messages.Domain.Entities.Message;
```

Comment:
Why do you rename it?

Reply:
It was necessary at some point, but not anymore.